core/web/router.go

```diff
@@ -364,11 +364,17 @@ func v2Routes(app chainlink.Application, r *gin.RouterGroup) {
 		authv2.PATCH("/chains/evm/:ID", chc.Update)
 		authv2.DELETE("/chains/evm/:ID", chc.Delete)
 
-		nc := NodesController{app}
-		authv2.GET("/nodes", paginatedRequest(nc.Index))
-		authv2.GET("/chains/evm/:ID/nodes", paginatedRequest(nc.Index))
-		authv2.POST("/nodes", nc.Create)
-		authv2.DELETE("/nodes/:ID", nc.Delete)
+		enc := EVMNodesController{app}
+		authv2.GET("/nodes/evm", paginatedRequest(enc.Index))
```

Comment:
Hmm, this could potentially break some clients...

- Should we bump the API version?
- Can we support both `/nodes` and `/nodes/evm` to stay backward compatible?

Reply:
I was thinking we had just done the equivalent with `/chains`->`/chains/evm`, but that was actually new when added. From a UX persepective it could make sense for the top level `/nodes` to list the common fields from all types, but if we need to stay compatible then no way around that :shrug:

Reply:
Restored so legacy paths (and ui) are EVM only for now, with a follow up story: https://app.shortcut.com/chainlinklabs/story/26276/multi-chain-type-ui-node-chain-configuration